hack/install-vm.sh

```diff
@@ -0,0 +1,407 @@
+        - /run/xtables.lock:/run/xtables.lock
+    environment:
+        - NODE_NAME=$NODE_NAME
+    command: $ANTREA_AGENT_BIN --config /etc/antrea/antrea-agent.conf --logtostderr=false --log_file=/var/log/antrea/antrea-agent.log
+    network_mode: host
+    privileged: true
+    cap_add:
+        - SYS_MODULE
+EOF
+}
+​
+load_ovs_modules() {
+    echo "Loading OVS kernel"
+    for module in "vport_geneve" "openvswitch"; do
+        lsmod | grep -q $module
+        if [[ $? -ne 0 ]]; then
+            echo "Load $module kernel module"
+            modprobe $module
+        else
+            echo "kernel module $module already loaded"
+        fi
+    done
+}
+​
+# Function to do the required job for starting antrea specfic containers
+start_containers() {
+    set +eo pipefail
+    echo "Extract Docker image from tar file"
+    docker pull $ANTREA_DOCKER_IMAGE
+    #docker load < $AGENT_CONF_PATH/$ANTREA_DOCKER_IMAGE_TAR
+    if [[ "$?" -ne 0 ]]; then
+        echoerr "ERROR!!docker pull from tar file failed"
+        exit 2
+    fi
+    # Load ovs kernel modules
+    load_ovs_modules
+    echo "$(date): Executing docker-compose up"
+    docker-compose -f $AGENT_CONF_PATH/docker-compose.yaml up -d
+
+    set -eo pipefail
+    # Start vm-watchdog systemd service to start/monitor containers
+    #create_antrea_agent_watchdog_service
+}
+​
+validate_argument() {
+    if [[ $2 == --* || -z $2 ]]; then
+        echoerr "Error invalid argument for $1: <$2>"
+        print_usage
+        exit 1
+    fi
+}
+​
+while [[ $# -gt 0 ]]
+do
+key="$1"
+case $key in
+    --install-docker)
+    INSTALL_DOCKER=$2
+    validate_argument $1 $2
+    shift 2
+    ;;
+    --ns)
+    AGENT_NAMESPACE="$2"
+    validate_argument $1 $2
+    shift 2
+    ;;
+    --bin)
+    AGENT_BIN_PATH="$2"
+    validate_argument $1 $2
+    shift 2
+    ;;
+    --config)
+    CONFIG_PATH="$2"
+    validate_argument $1 $2
+    shift 2
+    ;;
+    --kubeconfig)
+    KUBECONFIG="$2"
+    validate_argument $1 $2
+    shift 2
+    ;;
+    --antrea-kubeconfig)
+    ANTREA_KUBECONFIG="$2"
+    validate_argument $1 $2
+    shift 2
+    ;;
+    --nodename)
+    NODE_NAME="$2"
+    validate_argument $1, $2
+    shift 2
+    ;;
+    --ovs-bridge)
+    OVS_BRIDGE="$2"
+    shift 2
+    ;;
+    --validate-ovs)
+    VALIDATE_OVS_CONFIG=true
+    shift 1
+    ;;
+    -h|--help)
+    print_usage
+    exit 0
+    ;;
+    *)    # unknown option
+    echoerr "Unknown option $1"
+    exit 1
+    ;;
+esac
+done
+​
+check_ovs_config_and_cleanup
+validate_mandatory_arguments
+check_supported_platform
+copy_antrea_agent_files
+update_antrea_agent_conf
+if [ "$INSTALL_DOCKER" = false ]; then
+    # Install required packages and start agents
+    start_antrea_agent_service
+else
+    install_docker_requisite_packages
+    # Create docker-compose file with container/service details
+    create_docker_compose_file
+    # Start services required for running ovs and antrea agent
```

Comment:
remove the comment. It is not correct description. Instead give an appropriate function name